Index: ../../../Nhap_mon_lap_trinh/Bai_tap_ve_nha/On_Thi/Bai_Tap/Bai_Tap/Matrix.cs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../../../Nhap_mon_lap_trinh/Bai_tap_ve_nha/On_Thi/Bai_Tap/Bai_Tap/Matrix.cs b/../../../Nhap_mon_lap_trinh/Bai_tap_ve_nha/On_Thi/Bai_Tap/Bai_Tap/Matrix.cs
new file mode 100644
--- /dev/null	
+++ b/../../../Nhap_mon_lap_trinh/Bai_tap_ve_nha/On_Thi/Bai_Tap/Bai_Tap/Matrix.cs	
@@ -0,0 +1,25 @@
+using System;
+using System.Reflection.Metadata;
+
+namespace Bai_Tap
+{
+    public class Matrix
+    {
+        public static int[,] NewMatrix(int rownNo, int columnNo)
+        {
+            Console.WriteLine("Tạo dữ liệu ma trận mới 2 chiều");
+            Console.WriteLine("Nhap so cot ");
+            int[,] newMatrix = new int[rownNo,columnNo];
+            int count = 0;
+            for (int i = 0; i < newMatrix.GetLength(0); i++)
+            {
+                for (int j = 0; j < newMatrix.GetLength(1); j++)
+                {
+                    Console.Write($"Nhap du lieu vi tri [{i+1},{j+1}]:");
+                    newMatrix[i, j] = int.Parse(Console.ReadLine());
+                }
+            }
+            return newMatrix;
+        }
+    }
+}
\ No newline at end of file
Index: Dockerfile
===================================================================
diff --git a/Dockerfile b/Dockerfile
new file mode 100644
--- /dev/null	
+++ b/Dockerfile	
@@ -0,0 +1,20 @@
+FROM mcr.microsoft.com/dotnet/aspnet:5.0 AS base
+WORKDIR /app
+EXPOSE 80
+EXPOSE 443
+
+FROM mcr.microsoft.com/dotnet/sdk:5.0 AS build
+WORKDIR /src
+COPY ["WebApplication1.csproj", "./"]
+RUN dotnet restore "WebApplication1.csproj"
+COPY . .
+WORKDIR "/src/"
+RUN dotnet build "WebApplication1.csproj" -c Release -o /app/build
+
+FROM build AS publish
+RUN dotnet publish "WebApplication1.csproj" -c Release -o /app/publish
+
+FROM base AS final
+WORKDIR /app
+COPY --from=publish /app/publish .
+ENTRYPOINT ["dotnet", "WebApplication1.dll"]
Index: .dockerignore
===================================================================
diff --git a/.dockerignore b/.dockerignore
new file mode 100644
--- /dev/null	
+++ b/.dockerignore	
@@ -0,0 +1,25 @@
+**/.dockerignore
+**/.env
+**/.git
+**/.gitignore
+**/.project
+**/.settings
+**/.toolstarget
+**/.vs
+**/.vscode
+**/.idea
+**/*.*proj.user
+**/*.dbmdl
+**/*.jfm
+**/azds.yaml
+**/bin
+**/charts
+**/docker-compose*
+**/Dockerfile*
+**/node_modules
+**/npm-debug.log
+**/obj
+**/secrets.dev.yaml
+**/values.dev.yaml
+LICENSE
+README.md
\ No newline at end of file
